content/developers/apps/about-apps.md

```diff
@@ -14,7 +14,7 @@ versions:
 
 Apps on {% data variables.product.prodname_dotcom %} allow you to automate and improve your workflow. You can build apps to improve your workflow.{% if currentVersion == "free-pro-team@latest" %} You can also share or sell apps in [{% data variables.product.prodname_marketplace %}](https://github.com/marketplace). To learn how to list an app on {% data variables.product.prodname_marketplace %}, see "[Getting started with GitHub Marketplace](/marketplace/getting-started/)."{% endif %}
 
-{% data reusables.marketplace.github_apps_preferred %}, but GitHub supports both {% data variables.product.prodname_oauth_app %}s and {% data variables.product.prodname_github_apps %}. For information on choosing a type of app, see "[About apps](/apps/about-apps/)" and "[Differences between apps](/apps/differences-between-apps/)."
+{% data reusables.marketplace.github_apps_preferred %}, but GitHub supports both {% data variables.product.prodname_oauth_app %}s and {% data variables.product.prodname_github_apps %}. For information on choosing a type of app, see "[Differences between GitHub Apps and OAuth Apps](/apps/differences-between-github-apps-and-oauth-apps/)."
```

Comment:
```suggestion
{% data reusables.marketplace.github_apps_preferred %}, but GitHub supports both {% data variables.product.prodname_oauth_app %}s and {% data variables.product.prodname_github_apps %}. For information on choosing a type of app, see "[Differences between GitHub Apps and OAuth Apps](/developers/github-marketplace/about-github-marketplace)."
```

Thanks so much for opening a PR with this fix @lucalves! When I go to the staged article https://docs-4590--hotfixabout-app-lin.herokuapp.com/en/developers/apps/about-apps and click on the new, it gives an error. 

I think we'll need to update the relative link to `/developers/github-marketplace/about-github-marketplace`. If you look up at line 7 in this diff, you can see that the old link was redirecting to the current article, so I think this updated link should work!

Let me know when you accept this change and we can double check that the link works and get this merged in 🎉